### PR TITLE
(doc) Add release notes for Agent 2.2.0

### PR DESCRIPTION
**Pull request by AdmiringWorm**
## Description Of Changes

This updates the release notes section for Chocolatey Agent to include
the changes that has gone into the v2.2.0 version.

## Motivation and Context

We want to include the release notes for any release created for Chocolatey Agent.

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left-hand side)/
* [ ] Menu structure has been updated

## Related Issue

N/A

### CODE REVIEW

**Comment by pauby** [comment]
LGTM!

File: src/content/docs/en-us/agent/release-notes.mdx [hunk 1]
```diff
@@ -35,6 +35,16 @@ This covers the release notes for the Chocolatey Agent Service (`chocolatey-agen
 
 <ChocolateyComponentDependenciesLink />
 
+## 2.2.0 (December 4, 2024) \{#v2.2.2}
```
Comment (pauby) [inline src/content/docs/en-us/agent/release-notes.mdx:38]:
```suggestion
## 2.2.0 (December 4, 2024) \{#v2.2.0}
```

Comment (pauby) [reply src/content/docs/en-us/agent/release-notes.mdx:38]: I think this was incorrect?

**Review by pauby — CHANGES_REQUESTED** [review]

File: src/content/docs/en-us/agent/release-notes.mdx (hunk 1 above)
Comment (AdmiringWorm) [reply src/content/docs/en-us/agent/release-notes.mdx:38]: Yes, you are correct. I'll get that fixed up.

**Review by pauby — APPROVED** [review]

**Pull request merged**